spec/classes/init_old_spec.rb

```diff
@@ -0,0 +1,1017 @@
+require 'spec_helper'
```

Comment:
Should this file be removed since it includes old in its name?

Reply:
Wasn't brave enough to remove the old tests yet.